README.md

```diff
@@ -4,7 +4,16 @@ This is a Nextflow pipeline for generating sequencing reports for the SNP&amp;Se
 ## Pre-requisites
 You need to:
   - install Nextflow (e.g. using conda `conda create -n nextflow-env nextflow` or downloading from [nextflow.io](https://www.nextflow.io/)).
+  - Make Nextflow binary executable:
```

Comment:
This is only applies when the latter option (download from nextflow) is used, also the nextflow binary isn't necessarily downloaded to `/usr/local/bin/nextflow`